src/main/java/space/obminyashka/items_exchange/util/ResponseMessagesHandler.java

```diff
@@ -72,5 +72,6 @@ public class PositiveMessage {
         public final String ACCOUNT_ACTIVE_AGAIN = "account.made.active.again";
         public final String EMAIL_REGISTRATION_HEADER = "registration.header";
         public final String EMAIL_REGISTRATION_BODY = "registration.body";
+        public final String EMAIL_CONFIRMATION_MESSAGE = "confirmation.message";
```

Comment:
You shouldn't use **message** either in the constant name or the key itself. It would be better to rename it to **EMAIL_CONFIRMED = "email.confirmed"**